src/components/common/ColonyActionsTable/partials/ActionBadge/ActionBadge.tsx

```diff
@@ -0,0 +1,39 @@
+import clsx from 'clsx';
+import React, { type FC } from 'react';
+
+import { MotionState } from '~utils/colonyMotions.ts';
+import { useGetExpenditureData } from '~v5/common/ActionSidebar/hooks/useGetExpenditureData.ts';
+import ExpenditureBadge from '~v5/common/ActionSidebar/partials/ExpenditureBadge/index.ts';
+import MotionStateBadge from '~v5/common/Pills/MotionStateBadge/index.ts';
```

Comment:
Please import the file directly, and remove the use of the index barrel file

```suggestion
import ExpenditureBadge from '~v5/common/ActionSidebar/partials/ExpenditureBadge/ExpenditureBadge.tsx';
import MotionStateBadge from '~v5/common/Pills/MotionStateBadge/MotionStateBadge.tsx';
```